samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -100,20 +119,40 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
               />
             )}
             {
-              /* @conditional-compile-remove(PSTN-calls) */ outBoundCallChosen && (
+              /* @conditional-compile-remove(1-n-calling) */ acsCallChosen && (
                 <Stack>
-                  <TextField
-                    className={outboundTextField}
-                    label={'Participants'}
-                    placeholder={"Comma seperated phone numbers or ACS ID's"}
-                    onChange={(_, newValue) => newValue && setOutboundParticipants(newValue)}
-                  />
-                  <TextField
-                    className={outboundTextField}
-                    label={'ACS phone number for Caller ID'}
-                    placeholder={'Enter your ACS aquired phone number for PSTN call'}
-                    onChange={(_, newValue) => newValue && setAlternateCallerId(newValue)}
-                  />
+                  <Stack>
+                    <TextField
+                      className={outboundTextField}
+                      label={'Participants'}
+                      placeholder={"Comma seperated ACS user ID's"}
+                      onChange={(_, newValue) => setOutboundParticipants(newValue)}
+                    />
+                  </Stack>
+                </Stack>
+              )
+            }
+            {
+              /* @conditional-compile-remove(PSTN-calls) */ pstnCallChosen && (
+                <Stack>
```

Comment:
Why the 2 stacks? Seem redundant

Reply:
Hold over from previous version, will remove